corehq/apps/userreports/const.py

```diff
@@ -88,4 +88,4 @@
     (UCR_NAMED_FILTER, UCR_NAMED_FILTER),
 )
 
-DATA_SOURCE_REBUILD_RESTRICTED_AT = 1000000
+DATA_SOURCE_REBUILD_RESTRICTED_AT = 1000
```

Comment:
This is a temporary change for easier testing. Will drop the commit so the limit is updated back to 1M.